lib/miq_report_formats.rb

```diff
@@ -0,0 +1,44 @@
+class MiqReportFormats
```

Comment:
That was first thought as well.

Then I saw this class more as library than a model. And then, seeing lib/miq_expression depends on it I become convinced.

If you see MiqReportFormats more as model then library, I can rework this.